src/main/java/core/basesyntax/operations/BalanceOperation.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.operations;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.handler.OperationHandler;
+import core.basesyntax.model.FruitTransaction;
+
+public class BalanceOperation implements OperationHandler {
+    @Override
+    public void apply(FruitTransaction transaction) {
+        String fruits = transaction.getFruit();
```

Comment:
The variable `fruits` should be renamed to `fruit` to better reflect that it represents a single fruit name, not multiple fruits.